dip-0009/assignments.md

```diff
@@ -6,6 +6,7 @@ Here is a table of current feature paths and any associated DIP. Future DIPs may
 | Feature Index * | Feature | DIP Number and Name | Note |
 | ------------------ | ------------ | ------------------- | ---- |
 | `3'` | Masternode Keys | [DIP 0003: Deterministic Masternode List](https://github.com/dashpay/dips/blob/master/dip-0003.md) | The masternode related keys are located in the following sub-paths:  <br>`0'/*` - _Reserved_<br>`1'/*` - Voting Key<br>`2'/*` - Owner Key<br>`3'/*` - Operator Key<br>`4'/*` - [Platform P2P Key](../dip-0028.md#calculating-the-platform-node-id) (ED25519)<br><br>For example, the first voting key for Dash would be at `m/9'/5'/3'/1'/0` |
+| `4'` | CoinJoin Keys | CoinJoin | The masternode related keys are located in the following sub-paths:  <br>`0'/0/*` - Account 0 (external)<br>`0'/1/*` - Account 0 (change, not typically used)<br><br>For example, the first key of Account 0 for CoinJoin would be at `m/9'/5'/4'/0'/0/0`.<br><br>The main purpose of this using this path is to segregate CoinJoin related keys from the BIP44 that are used to receive coins and change.  CoinJoin derivation paths may require different lookup or gap values than the BIP44 paths that have much less transaction activity.  Additionally, it may be possible to prune used keys from the BIP37 bloom filters used by mobile apps.|
```

Comment:
I think there's a missing word. Not sure if it should be paths or keys.
```suggestion
| `4'` | CoinJoin Keys | CoinJoin | The masternode related keys are located in the following sub-paths:  <br>`0'/0/*` - Account 0 (external)<br>`0'/1/*` - Account 0 (change, not typically used)<br><br>For example, the first key of Account 0 for CoinJoin would be at `m/9'/5'/4'/0'/0/0`.<br><br>The main purpose of this using this path is to segregate CoinJoin-related keys from the BIP44 paths that are used to receive coins and change.  CoinJoin derivation paths may require different lookup or gap values than the BIP44 paths that have much less transaction activity.  Additionally, it may be possible to prune used keys from the BIP37 bloom filters used by mobile apps.|
```